citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ConceptpowerImporter.java

```diff
@@ -124,6 +124,7 @@ public AuthoritySearchResult searchAuthorities(String firstName, String lastName
                     IAuthorityEntry authority = new AuthorityEntry();
                     authority.setName(conceptEntry.getLemma());
                     authority.setUri(conceptEntry.getConcept_uri());
+                    authority.setSource("Conceptpower");
```

Comment:
this should not be hard-coded but a configuration property

Reply:
actually, this should be the importer id and then the human-friendly version would be in the config file mapped to the id.